extensions/wikia/VideoEmbedTool/js/VET_Loader.js

```diff
@@ -84,12 +84,15 @@
 			return;
 		} else if (window.wgUserName === null) {
 			// handle login on article page
-			window.UserLoginModal.show({
-				origin: 'vet',
-				callback: function () {
-					window.UserLogin.forceLoggedIn = true;
-					vetLoader.load(options);
-				}
+			require(['AuthModal'], function (authModal) {
+				authModal.load({
+					url: '/signin',
+					origin: 'vet',
+					successAuthCallback: function () {
+						window.UserLogin.forceLoggedIn = true;
```

Comment:
It is checked in a few places later. I tried not to change existing logic with implementing the new auth modals. On the other hand, "forceLoggedIn" have more sense to be set to true "after" user is authenticated than it would have before authentication.